autopush/tests/test_web_validation.py

```diff
@@ -158,13 +158,13 @@ def _make_fut(self):
         schema.context["log"] = Mock()
         return schema
 
-    def _make_test_data(self, headers=None, body="", path_args=None,
-                        path_kwargs=None, arguments=None):
+    def _make_test_data(self, headers=None, body="", args=None, kwargs=None,
+                        arguments=None):
```

Comment:
Yeah, like I said, a bit more nitty than needed. I'm also fine with just dropping "args" since I don't think that anything actually passes them. Calling "arguments" `request_arguments` might be good. Granted, shortening down 'kwargs' and tossing it into the middle of the arg list might cause a few python folk to twitch, but none of that is super critical.